src/Type/Definition/BooleanType.php

```diff
@@ -14,33 +13,22 @@
 
 class BooleanType extends ScalarType
 {
-    /** @var string */
     public $name = Type::BOOLEAN;
```

Comment:
why not do this so it's type safe?

```suggestion
    public string $name = Type::BOOLEAN;
```

Reply:
See https://github.com/webonyx/graphql-php/pull/906